shards/modules/anim/anim.cpp

```diff
@@ -173,17 +180,17 @@ struct TimerShard {
 };
 
 struct PlayShard {
-  static SHOptionalString help() { return SHCCSTR(R"(Outputs the duration of an animation, in seconds)"); }
+  static SHOptionalString help() { return SHCCSTR("Plays the animation of the glTF model specified in the Animation parameter."); }
```

Comment:
I'd say something like "evaluates the animation at the given time".
Since it technically doesn't "Play" the animation, just interpolates all the timelines in the animation and gives you the output, which you then have to apply to the mesh to actually "play"